crypto/src/bls12377/mod.rs

```diff
@@ -483,9 +483,9 @@ impl Default for BLS12377AggregateSignature {
 }
 
 impl AggregateAuthenticator for BLS12377AggregateSignature {
-    type PrivKey = BLS12377PrivateKey;
-    type PubKey = BLS12377PublicKey;
     type Sig = BLS12377Signature;
+    type PubKey = BLS12377PublicKey;
```

Comment:
We should maintain the members order according to trait for consistency and readability.